docs/UserGuide.md

```diff
@@ -181,88 +165,90 @@ Format: **`help`**
 
 ### Adding a person: `add`
 
-If you want to add a new person into Socius, you can use the `add` command.
+You can add a new person into Socius with the `add` command.
 
 Format: **`add`** `n/NAME [p/PHONE_NUMBER] [e/EMAIL] [nat/NATIONALITY] [g/GENDER] [tg/TUTORIAL_GROUP] [s/SOCIAL_HANDLE]…​ [r/REMARK] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 The [Parameter Constraints](#parameter-constraints) section shows the constraints for each parameter.
 </div>
 
-* Only the `NAME` field of a person is _compulsory_, other fields are _optional_.
-* The order of the fields does not matter.
+* Only the `NAME` parameter of a person is _compulsory_, other parameters are _optional_.
+* The parameters can be in any order.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0) and social handles (including 0)
+A person can have any number of tags and social handles (including 0)
 </div>
 
 Examples:
 
 * **`add`** `n/Amy Tan` adds `Amy Tan` to the list.
-* **`add`** `n/Benedict p/98765432 e/ben@example.com g/M` adds `Benedict`  to the list together with his phone number, email and gender.
-* **`add`** `n/Ernest s/tg:ernest2334 s/ig:ernessst` adds `Ernest` to the list together with his Telegram and Instagram handles.
+* **`add`** `n/Benedict p/98765432 e/ben@example.com g/M` adds `Benedict`, a `Male` with phone number `98765432` and email `ben@yahoo.com`  to your contacts list.
```

Comment:
- There seems to be an extra space after `ben@yahoo.com`.
- The email does not tally
- Perhaps avoid using `contacts list` or perhaps leave a note somewhere to define what `contacts list` is, so that user will not get confused with the term `Socius`, `address book` and `contacts list`
- I am not sure if it is `contact list` or `contacts list`

Reply:
Corrected the email issues.
I have standardised by switching 'address book' with 'contacts list', cos address book sounds like Yellow Pages.
Change to 'contact list'